docker/develop/develop.dockerfile

```diff
@@ -1,12 +1,31 @@
 # vim: ft=dockerfile
+# Define the names/tags of the container
+#!BuildTag: cobbler-test-github:release33 cobbler-test-github:release33.%RELEASE%
+
+# We don't want to version pin our dependencies for testing. Always retrieve what is up to date.
+# hadolint global ignore=DL3037
 
 # WARNING! This is not in any way production ready. It is just for testing!
-FROM registry.opensuse.org/opensuse/leap:15.3
+FROM opensuse/leap:15.3
```

Comment:
Shouldn't we use a still supported Leap version?

Reply:
I kept the base image from when this Cobbler version was released. Of course, it is possible upgrading the CI but it is a change that potentially can cause trouble due to the change in dependencies between the different Leap versions. If you insist I can upgrade the base image however, I would like to do this in a follow-up PR.